pkg/virt-api/webhooks/arm64.go

```diff
@@ -0,0 +1,157 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021
+ *
+ */
+
+/*
+ * arm64 utilities are in the webhooks package because they are used both
+ * by validation and mutation webhooks.
+ */
+package webhooks
+
+import (
+	"fmt"
+	rt "runtime"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sfield "k8s.io/apimachinery/pkg/util/validation/field"
+
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+var _false bool = false
+var VerifyArchFunc = defaultVerifyArchFunc
```

Comment:
I know it sounds convenient at first to mock things this way out, but please consider using function arguments to pass the architecture or change parts of the weebhook code to structs. These hidden shared globals cause me a lot of trouble on refactorings.

Reply:
Ok, I get it.